env/CMakeLists.txt

```diff
@@ -2,7 +2,8 @@ cmake_minimum_required(VERSION 3.20.0)
 project(ttmlir-toolchain LANGUAGES CXX C)
 
 set(FLATBUFFERS_VERSION "fb9afbafc7dfe226b9db54d4923bfb8839635274")
-set(LLVM_PROJECT_VERSION "9ddfe62f5c11e3f65f444209f514029ded2d58b9")
+set(LLVM_PROJECT_VERSION "e813750354bbc08551cf23ff559a54b4a9ea1f29")
```

Comment:
Can you just varify this test doesn't fail (but you have to sync to main if that is not a problem to you?):

`/home/mtopalovic/src/tt-mlir/build/bin/ttmlir-opt --ttir-to-ttnn-backend-pipeline /localdev/mtopalovic/src/tt-mlir/test/ttmlir/Dialect/TTNN/simple_broadcast.mlir`

Reply:
Sure, but how will uplifting the LLVM version have impact on this test?

Reply:
Ok I can confirm that remove-dead-values works after uplifting  LLVM to this version and the above test passes.